docs/site/content/docs/getting-started/build-cloud-image.md

```diff
@@ -9,86 +9,231 @@ sort_by = "weight"
 template = "docs/page.html"
 
 [extra]
-lead = "One page summary of how to start a new AdiDoks project."
```

Comment:
![image](https://user-images.githubusercontent.com/8912557/135044112-ba1f560a-266f-4d8b-97c9-ec2f7245c460.png)
Change this description